app/backend/src/couchers/models/notifications.py

```diff
@@ -251,6 +252,26 @@ class NotificationDelivery(Base):
     )
 
 
+class DeviceType(enum.Enum):
+    ios = enum.auto()
+    android = enum.auto()
+
+
+class PushNotificationPlatform(enum.Enum):
+    web_push = enum.auto()
+    expo = enum.auto()
+
+
+class PushNotificationDeliveryOutcome(enum.Enum):
+    success = enum.auto()
```

Comment:
Should we just make success a boolean? Then have "error_reason" that is the strings? Then we don't need to compare strings when handling errors.